ItemRepeater.cs

```diff
@@ -99,6 +99,10 @@ public override bool Run(IssueManager issueManager)
 
                         int issueId = issue.Id;
 
+                        //Set the ID to 0 so it does not steal the custom fields for that tissue
+                        var originalIssueID = issue.Entity.Id;
+                        issue.Entity.Id = 0;
```

Comment:
Please check if I have missed anything on this. Not sure why we're creating a new issue with an already existing ID. This was stealing the CFs from the old issue since the convert method loaded the related info based on the ID. Rather than a new list of CFs it loaded the one from the other ID, even though the item being created is brand new and needs the new defaults (ready to be updated later)